patches/nan/api_simplify_scriptorigin.patch

```diff
@@ -0,0 +1,20 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: John Kleinschmidt <jkleinsc@github.com>
+Date: Wed, 2 Dec 2020 11:19:26 -0500
+Subject: [api] Simplify ScriptOrigin
+
+Update nantest to handle changes in  https://chromium-review.googlesource.com/c/v8/v8/+/2555005
+
+diff --git a/test/cpp/nannew.cpp b/test/cpp/nannew.cpp
+index a489f9405d3d39dfb71375d490ba32f9c05c610d..c6cca837f485ac4a245bd2d4a479e33f6250fd1d 100644
+--- a/test/cpp/nannew.cpp
++++ b/test/cpp/nannew.cpp
+@@ -246,7 +246,7 @@ NAN_METHOD(testScript) {
+ 
+   t.plan(6);
+ 
+-  ScriptOrigin origin(New("foo").ToLocalChecked(), New(5));
++  ScriptOrigin origin(New("foo").ToLocalChecked(), 5);
```

Comment:
Does this need to be upstreamed to Nan?

Reply:
Definitely, patching `nan` here gets us absolutely nothing.  The point of the `nan` tests is to detect when we need to fix things upstream.  Changing the test isn't acceptable either, we need to find a way to make this work in nan itself as it needs to retain backwards compatibility.

Reply:
Yes - i can open an upstream PR